package.json

```diff
@@ -32,13 +32,13 @@
 	"devDependencies": {
 		"@automattic/color-studio": "^2.5.0",
 		"@automattic/wp-babel-makepot": "^1.2.0",
-		"@electron-forge/cli": "^7.4.0",
-		"@electron-forge/maker-deb": "^7.4.0",
-		"@electron-forge/maker-dmg": "^7.4.0",
-		"@electron-forge/maker-squirrel": "^7.4.0",
-		"@electron-forge/maker-zip": "^7.4.0",
-		"@electron-forge/plugin-auto-unpack-natives": "^7.4.0",
-		"@electron-forge/plugin-webpack": "^7.4.0",
+		"@electron-forge/cli": "7.3.1",
```

Comment:
Note that I removed `^` from the version to avoid upgrading it to the minor version `7.4.0`. If we'd like to allow upgrading patch version we could replace it with `~`.